src/main/java/sirius/kernel/commons/Value.java

```diff
@@ -604,14 +604,33 @@ public Value first() {
      * or the <tt>defaultValue</tt> otherwise
      */
     @SuppressWarnings("unchecked")
-    public <V> V get(Class<V> clazz, V defaultValue) {
+    @Nonnull
+    public <V> V get(Class<V> clazz, @Nonnull V defaultValue) {
         Object result = get(defaultValue);
         if (!clazz.isAssignableFrom(result.getClass())) {
             return defaultValue;
         }
         return (V) result;
     }
 
+    /**
+     * Returns the wrapped value if it is an instance of the given clazz or <tt>null</tt> otherwise.
+     *
+     * @param clazz the desired class of the return type
+     * @param <V>   the expected type of the wrapped value
+     * @return the wrapped value if the given <tt>clazz</tt> is assignable from wrapped values class or <tt>null</tt>
+     * otherwise
+     */
+    @Nullable
+    @SuppressWarnings("unchecked")
+    public <V> V get(Class<V> clazz) {
+        Object result = get();
+        if (result == null || !clazz.isAssignableFrom(result.getClass())) {
```

Comment:
why not just add the nullcheck to the old method?

Reply:
nope, this is as intended. when you provide a default value you dont want the returning value to be null, so passing null would go against the point of the API. To provide methods where null as a return type is wanted and accepted a new method is added.